CODEOWNERS

```diff
@@ -29,6 +29,7 @@ src/audio/dcblock*			@cujomalainey @dgreid
 src/audio/crossover*			@cujomalainey @dgreid
```

Comment:
@johnylin76 care to add to the commit message the same explanation you added to the PR description? PR description is not saved in git history so it will get lost.

Reply:
Done. Thanks.